src/vector/index.js

```diff
@@ -136,9 +140,13 @@ window.onload = function() {
 
 function loadApp() {
     if (validBrowser) {
-        var MatrixChat = sdk.getComponent('pages.MatrixChat');
+        var MatrixChat = sdk.getComponent('structures.MatrixChat');
         window.matrixChat = ReactDOM.render(
-            <MatrixChat onNewScreen={onNewScreen} registrationUrl={makeRegistrationUrl()} />,
+            <MatrixChat
+                onNewScreen={onNewScreen}
+                registrationUrl={makeRegistrationUrl()}
+                ConferenceHandler={VectorConferenceHandler}
```

Comment:
Hmm, we have one system for customising and swapping out bits of the react sdk, but we're doing conference handling by injecting at the top level. This feels wrong.